server/routes/incidentio/tasks.py

```
@@ -0,0 +1,413 @@
+                                source_alert_id=alert_db_id,
+                                alert_title=fields["incident_name"],
+                                alert_service=service,
+                                alert_severity=normalized_severity,
+                                correlation_result=correlation_result,
+                                alert_metadata=alert_metadata,
+                                raw_payload=payload,
+                                org_id=org_id,
+                            )
+                            conn.commit()
+                            return
+                    except Exception as corr_exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)